CONTRIBUTING.rst

```diff
@@ -3,6 +3,13 @@
 Contributor Guide
 =================
 
+.. note::
+   This document assumes some familiarity with contributing to open source
+   scientific Python projects using GitHub pull requests. If this does not
+   describe you, you may first want to see the :ref:`contributing_faq`.
```

Comment:
Should this reference be to `new_contributor_faq` now instead of `contributing_faq`?

Reply:
In this case the `contributing_faq` is an explicitly specified link target (see the `.. _contributing_faq:` on the first line of `new_contributor_faq.rst` doc), so this link should work (and continue to work) regardless of the name of the file or the title of the document.